system_test/common/helpers/aest_external.erl

```diff
@@ -57,6 +58,8 @@ stop_node(NodeState, _Opts) -> NodeState.
 kill_node(_NodeState) ->
     error({not_supported, ?FUNCTION_NAME}).
 
+stop_container(NodeState, _Opts) -> NodeState.
```

Comment:
This seems not fitting the `aest_nodes` abstraction as it assumes container backend. E.g. it makes no sense for `aest_external`. At least it shall return `not_supported` here?
```suggestion
stop_container(NodeState, _Opts) ->
    error({not_supported, ?FUNCTION_NAME}).
```